pkg/model/interface.go

```diff
@@ -7,12 +7,13 @@ import (
 )
 
 type Model interface {
-	GetInferenceParameters() *PresetInferenceParam
+	GetInferenceParameters() *PresetParam
+	GetTrainingParameters() *PresetParam
 	SupportDistributedInference() bool //If true, the model workload will be a StatefulSet, using the torch elastic runtime framework.
 }
 
-// PresetInferenceParam defines the preset inference parameters for a model.
-type PresetInferenceParam struct {
+// PresetParam defines the preset inference parameters for a model.
+type PresetParam struct {
```

Comment:
1) Move BaseCommand to be adjacent to ModelRunParams;
2) Rename DeploymentTimeout-> ImagepullTimeout;
3) refresh all comments to explicitly;
4) Add a supporttuning interface method, used by validation.
// Llama does not support.